<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassCard</class>
 <widget class="QWidget" name="ClassCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>180</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>150</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="HitDie">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="Stat1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Str</string>
   </property>
  </widget>
  <widget class="QLabel" name="Stat5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Wis</string>
   </property>
  </widget>
  <widget class="QLabel" name="Stat3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Con</string>
   </property>
  </widget>
  <widget class="QLabel" name="Stat6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cha</string>
   </property>
  </widget>
  <widget class="QLabel" name="Stat4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Int</string>
   </property>
  </widget>
  <widget class="QLabel" name="Stat2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Dex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stat Priority</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
